src/main/java/org/jenkinsci/plugins/fodupload/steps/FortifyDastPipeline.java

```diff
@@ -893,23 +897,27 @@ private void saveApiScanSettings(FilePath workspace, PrintStream printStream, Da
                     openApiRadioSource, sourceUrn, openApiKey);
 
         } else if (FodEnums.DastApiType.GraphQL.toString().equalsIgnoreCase(selectedApiType)) {
+            if(FodEnums.ApiSourceType.valueOf(graphQlRadioSource) == FodEnums.ApiSourceType.FileId) {
+                FilePath patchPayload = new FilePath(workspace, this.graphQLFilePath);
 
-            FilePath patchPayload = new FilePath(workspace, this.graphQLFilePath);
+                if (!patchPayload.exists()) {
 
-            if (!patchPayload.exists()) {
+                    printStream.printf("FilePath for the Payload not constructed for releaseId %s%n", releaseId);

```

Comment:
Check if the function that invokes this method has the catch block and the logger inside the catch, If so , please remove the Utils.logger from here to avoid dual entry

Reply:
No, It does not has try catch. Thanks!

Reply:
Please check the calling function at line number 1102 which call the SaveScanSetting function , which in turns call the saveApiScanSettings function , and there is catch block ,